pkg/apis/crd/v1alpha1/types.go

```diff
@@ -291,3 +291,38 @@ type TLSProtocol struct {
 	// SNI (Server Name Indication) indicates the server domain name in the TLS/SSL hello message.
 	SNI string `json:"sni,omitempty"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeLatencyMonitor is used to monitor the latency between nodes in a Kubernetes cluster. It is a singleton resource,
+// meaning only one instance of it can exist in the cluster.
+type NodeLatencyMonitor struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec NodeLatencyMonitorSpec `json:"spec"`
+}
+
+type NodeLatencyMonitorSpec struct {
+	// PingInterval specifies the interval between ping requests.
+	// Ping interval should be greater than or equal to 1s(one second).
+	// Defaults to "60". Valid time units are "s".
```

Comment:
`Valid time units are "s". ` sounds user is expected to input the unit, which is not the case.

Reply:
+1. Please remove this sentence and replace the first sentence with "PingInterval specifies the interval in seconds between ping requests."

Reply:
Thanks for the suggestion, I'll change the comments here.